vignettes/epidist.Rmd

```diff
@@ -156,7 +156,7 @@ Here we suppose that the interval is daily, meaning that only the date of the pr
 obs_cens <- obs |> observe_process()
 ```
 
-(ref:cens) Interval censoring of the primary and secondary event times obscures the delay times. While daily censoring is most common, `epidist` supports the primary and secondary events having other delay intervals.
+(ref:cens) Interval censoring of the primary and secondary event times obscures the delay times. Many exact delay times (which can take any value) have the same censored delay times (which are restricted to a number of days). While daily censoring is most common, `epidist` supports the primary and secondary events having other delay intervals.
```

Comment:
```suggestion
(ref:cens) Interval censoring of the primary and secondary event times obscures the delay times. A common example of this is when events are reported as daily aggregates. While daily censoring is most common, `epidist` supports the primary and secondary events having other delay intervals.
```

I don't think this comment was addressed. I've reworded. If you feel strongly about the previous sentence please clarify why.